interfaces/__template__/v0/interface.yaml

```diff
@@ -0,0 +1,33 @@
+# `name` of the interface, as it should be referenced by charms.
+# This should be the same as the name of the interface specification directory.
+# When proposing a new interface, opt for underscores over dashed.
+# The name corresponds to the line with the arrow in the charm `metadata.yaml` snippet below:
+#
+# ```yaml
+# provides:
+#   self-metrics-endpoint:
+#     interface: prometheus_scrape <--
+# ```
+#
+name: __template__
+
+# Uncomment the line below if the interface is to be considered internal, meaning that it
+# is not designed to be used by others, nor should it appear on charmhub:
+#
+# internal: true
+
+# `version` starts at `0` and increments with every breaking change in the interface specification.
+version: 0
+
+# `status` describes where the interface is in its lifecycle. Valid values are described in LEGEND.md.
```

Comment:
[LEGEND.md](LEGEND.md) is deleted in this PR, so I assume this should point people elsewhere?